app/src/main/kotlin/com/wire/android/model/ImageAsset.kt

```diff
@@ -72,53 +60,39 @@ sealed class ImageAsset {
             withCrossfadeAnimation: Boolean = false
         ) = when {
             LocalInspectionMode.current -> painterResource(id = R.drawable.ic_welcome_1)
-            else -> {
-                hiltViewModel<RemoteAssetImageViewModel>(
-                    // limit the scope of the ViewModel to the current activity so that there's one image loader instance for the Activity
-                    viewModelStoreOwner = checkNotNull(LocalActivity.current as? AppCompatActivity ?: LocalViewModelStoreOwner.current) {
-                        "No ViewModelStoreOwner was provided via LocalViewModelStoreOwner"
-                    },
-                    key = "remote_asset_image_loader"
-                ).imageLoader.paint(asset = this, fallbackData = fallbackData, withCrossfadeAnimation = withCrossfadeAnimation)
```

Comment:
I'm not sure but now after understanding the issues with images I think that this could be the problem, so just removing passing `viewmodelStoreOwner` and a `key` should make it so that this view model and image loader is created per particular screen and not whole activity which means it's created again when account is switched 🙇

Reply:
@yamilmedina I just checked that and images are loading fine for both logged in accounts from different federated environments after I removed just these two parameters `viewModelStoreOwner` and `key`, so it looks like this:
```
else -> hiltViewModel<RemoteAssetImageViewModel>().imageLoader
    .paint(asset = this, fallbackData = fallbackData, withCrossfadeAnimation = withCrossfadeAnimation)
```

Reply:
thanks for the fix